### PR TITLE
Release 1.6 prep

### PR DESCRIPTION
**Pull request by mtuncer**
updated extension control, sql files, and documentation regarding 1.6 version

### CODE REVIEW

**Comment by mtuncer** [comment]
@pykello ping

**Comment by pykello** [comment]
(My status is that I am testing this briefly before we merge it).

File: README.md [hunk 1]
```diff
@@ -284,11 +284,20 @@ the installation:
 
 Changeset
 ---------
+### Version 1.6
+* (Feature) Added support for PostgreSQL 10.
```
Comment (mtuncer) [inline README.md]: We would do 1.6.1 if that happens

File: .gitignore [hunk 2]
```diff
@@ -46,5 +46,12 @@
 /sql/create.sql
 /sql/data_types.sql
 /sql/load.sql
+/sql/copyto.sql
```
Comment (mtuncer) [inline .gitignore]: done

File: META.json [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
+   "version": "1.6.0",
    "maintainer": "Murat Tuncer <mtuncer@citusdata.com>",
    "license": "apache_2_0",
    "provides": {
       "cstore_fdw": {
          "abstract": "Foreign Data Wrapper for Columnar Store Tables",
-         "file": "cstore_fdw--1.5.sql",
+         "file": "cstore_fdw--1.6.sql",
          "docfile": "README.md",
          "version": "1.5.0"
```
Comment (pykello) [inline META.json]: 1.6.0

**Pull request merged**